src/Faker/Provider/fa_IR/Person.php

```diff
@@ -134,4 +134,82 @@ class Person extends \Faker\Provider\Person
 
     protected static $titleMale = array('آقای', 'استاد', 'دکتر', 'مهندس');
     protected static $titleFemale = array('خانم', 'استاد', 'دکتر', 'مهندس');
+
+    /**
+     * this method returns a valid Iranian nationalCode
```

Comment:
please add a link to an official specification of that code

Reply:
Added the reference link.